mobile/lib/presentation/widgets/action_buttons/add_action_button.widget.dart

```diff
@@ -0,0 +1,201 @@
+// dart
+// File: 'lib/presentation/widgets/action_buttons/add_action_button.widget.dart'
+import 'package:easy_localization/easy_localization.dart';
+import 'package:flutter/material.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+import 'package:immich_mobile/extensions/build_context_extensions.dart';
+import 'package:immich_mobile/presentation/widgets/action_buttons/base_action_button.widget.dart';
+import 'package:immich_mobile/providers/infrastructure/asset_viewer/current_asset.provider.dart';
+import 'package:immich_mobile/presentation/widgets/album/album_selector.widget.dart';
+import 'package:immich_mobile/providers/infrastructure/album.provider.dart';
+import 'package:immich_mobile/widgets/common/immich_toast.dart';
+import 'package:immich_mobile/providers/user.provider.dart';
+
+import 'package:immich_mobile/domain/models/album/album.model.dart';
+import 'package:immich_mobile/domain/models/asset/base_asset.model.dart';
+
+import '../../../constants/enums.dart';
+import 'archive_action_button.widget.dart';
+import 'move_to_lock_folder_action_button.widget.dart';
+
+enum _AddMenu { album, archive, lockedFolder }
+
+class AddActionButton extends ConsumerWidget {
+  const AddActionButton({super.key});
+
+  @override
+  Widget build(BuildContext context, WidgetRef ref) {
+    return Builder(
+      builder: (buttonContext) {
+        return BaseActionButton(
+          iconData: Icons.add,
+          label: "add_to".tr(),
```

Comment:
can you make a new translation key that is just "Add to" without the ellipses? You can call it "bottom_bar_add_to" so it doesn't conflict.